doc/classes/ProjectSettings.xml

```diff
@@ -60,6 +60,16 @@
 				Clears the whole configuration (not recommended, may break things).
 			</description>
 		</method>
+		<method name="get_autoload_list" qualifiers="const">
+			<return type="Dictionary[]" />
+			<description>
+				Returns an [Array] of registered autoload classes. Each class is represented as a [Dictionary] that contains the following entries:
+				- [code]name[/code] is the name for both the node and the global variable of the autoload;
+				- [code]path[/code] is the path to the autoloaded resource;
+				- [code]is_singleton[/code] is whether the autoload is registered as a global variable;
```

Comment:
```suggestion
				- [code]is_singleton[/code] is [code]true[/code] if the autoload can be accessed as a global variable, otherwise it is [code]false[/code].
```